src/components/NavBar.vue

```diff
@@ -48,6 +48,7 @@
                     <b-nav-item-dropdown text="Projects">
                         <b-dropdown-item to="/projects/covid19/">Covid19</b-dropdown-item>
                         <b-dropdown-item to="/projects/mpxv/">Monkeypox</b-dropdown-item>
+                        <b-dropdown-item to="/projects/vgp/">VGP</b-dropdown-item>
```

Comment:
Is the acronym widely known? If we spell it out, it will be too long, I guess...

Reply:
It seemed a bit long for the dropdown, but the top of the page is spelled out, so it's immediately visible once clicked, at least